pyproject.toml

```diff
@@ -10,7 +10,7 @@ requires-python = "<3.13,>=3.11"
 dependencies = [
     "fastapi>=0.112.0,<1",
     "httpx>=0.27.0,<1",
-    "mex-common @ git+https://github.com/robert-koch-institut/mex-common.git@0.34.0",
+    "mex-common @ git+https://github.com/robert-koch-institut/mex-common.git@feature/mx-1417-fix-temporal-serialization",
```

Comment:
Could you set the correct mex-common version?

Reply:
done [a8b3cf6](https://github.com/robert-koch-institut/mex-backend/pull/106/commits/a8b3cf611ea0e3f7aae7e20ef76ec93a1697dc1b)